.gitignore

```diff
@@ -4,4 +4,5 @@ npm-debug.log
 src/*.js
 src/*.js.map
 src/*.d.ts
-/compiled
\ No newline at end of file
+/compiled
+dist/
```

Comment:
You need to un-track `dist/` for this to take effect (`git rm -r --cached dist/` iirc)